src/routes/shows/+page.svelte

```diff
@@ -1,14 +1,28 @@
 <script lang="ts">
+	import { getShowsByVideoCount } from '$lib/data'
```

Comment:
Maybe the sorting should be handled by a function within this component rather than pulling in these functions from the `$lib/data` file? I do like the idea that we don't do any "data loading" in the component, but keep that to the `load()` functions.

So put logic that re-sorts the show list into the `getSortedShows()` function below.

Reply:
Good call!